### ISSUE TITLE
[HOLD on 41357] LHN - Admin and announcement rooms are archived when try to delete annual workspace

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#46538]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.14-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/4784660&group_by=cases:section_id&group_order=asc&group_id=229065
**Email or phone of affected tester (no customers):** gocemate+a779@gmail.com
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Log in with new hmail account
2. Go to Account settings> Workspace> Create workspace
3. Delete the workspace
4. Verify that error message "Looks like you're on an annual subscription. Please downgrade your account from your account settings before trying again." appear> Dismiss the error message
5. Go to LHN and check rooms status

## Expected Result:
Rooms should not be archived because the workspace is not deleted

## Actual Result:
Admin and announcement rooms are archived when try to delete annual workspace. When re-login the rooms are back to normal state

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/user-attachments/assets/fbca9e95-73e2-48fc-9416-4c73b867a99f

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
@RachCHopkins  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by RachCHopkins** [comment]
This is actually part of a larger issue where we should let people delete their workspace if they have not started / been billed for their subscription yet. Maybe we should solve that instead? Is that something that is front-end?

**Comment by RachCHopkins** [comment]
Raised in [Slack](https://expensify.slack.com/archives/C049HHMV9SM/p1722990568242669).

**Comment by RachCHopkins** [comment]
Ok, yes, this is due to [this bug](https://github.com/Expensify/App/issues/41357) and once that is fixed it should no longer be a problem. Let's hold and restest when that one is fixed.

**Comment by RachCHopkins** [comment]
Still waiting on 41357

**Comment by RachCHopkins** [comment]
Ditto

**Comment by RachCHopkins** [comment]
The other issue is on Hold. We can reopen this in future if it's needed after the other one is completed.
